src/App/Header/UserMenu/UserMenu.tsx

```diff
@@ -0,0 +1,24 @@
+import { Authenticator, useAuthenticator } from "@aws-amplify/ui-react";
+import { Popover } from "@headlessui/react";
+import Icon from "components/Icon";
+import Menu from "./Menu";
+
+export default function UserMenu() {
+  const { user, signOut } = useAuthenticator((context) => [context.route]);
+  return (
+    <Popover className="relative">
+      {user ? (
+        <Popover.Button className="cursor-pointer contents">
+          <Icon type="User" size={24} className="text-white" />
+        </Popover.Button>
+      ) : (
+        <Popover.Button className="btn-orange px-3 h-10 rounded-lg text-sm">
+          Login
```

Comment:
Suggestion: _we should use consistent naming_, i.e. if this is called "Login", then the opposite functionality should be called "Logout" (as opposed to the current "Sign out") or vice versa ("Sign in" + "Sign out").
Considering Amplify uses the latter terminology, it would make sense to use the same one

Reply:
Agreed! That's prob my fault in writing up the specs. :bow: "Sign up" and "Sign In" is most consistent. :+1:

Reply:
@misicnenad, @SovereignAndrey - done 
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/c654a1a7-910d-42bc-bdaf-f562c94718d2)